src/app/extensions/quoting/pages/quote-edit/quote-edit-page.component.ts

```diff
@@ -39,4 +50,25 @@ export class QuoteEditPageComponent implements OnInit {
   addQuoteToBasket(quoteId: string) {
     this.quotingFacade.addQuoteToBasket(quoteId);
   }
+
+  private patchBreadcrumbData() {
+    this.quote$
+      .pipe(
+        whenTruthy(),
+        withLatestFrom(this.appFacade.breadcrumbData$),
+        take(1)
+      )
+      .subscribe(([quote, breadcrumbData]) => {
+        this.translateService
+          .get(
```

Comment:
why didn't you use this.translateService.instant ?

Reply:
i prefer get over instant because its more save

> instant(key: string|Array<string>, interpolateParams?: Object): string|Object: Gets the instant translated value of a key (or an array of keys). /!\ This method is synchronous and the default file loader is asynchronous. You are responsible for knowing when your translations have been loaded and it is safe to use this method. If you are not sure then you should use the get method instead.
(https://github.com/ngx-translate/core)